yaml/simulator.yaml

```diff
@@ -135,3 +135,8 @@
   sim:
     cmd: >
       xrun -R -xmlibdirpath <out> <sim_opts> -svseed <seed> -svrnc rand_struct -nokey
+
+- tool: pyflow
+  sim:
+    cmd: >
+      python3
```

Comment:
I'm not quite sure we should leave this command in the YAML file as just `python3` - can we maybe do something like this?
```console
cmd: >
  python3 <cwd>/pygen/pygen_src/test/<test_name>.py <sim_opts> ...
```
where `<cwd>` is the root of the riscv-dv repo?

I think this way we can let the section on line 278 of `run.py` in this PR just deal with assembling the cwd, test_name, any external sim_opts, and the other 4 'default' arguments - while still making it clear to anyone looking at `simulator.yaml` what the general idea of the Python tool command is doing. Wdyt?

Reply:
I actually also thought of it, having just "python3" as the sim cmd in the `simulator.yaml` doesn't look good, and it also doesn't provide enough information about this simulation method. I agree with you, and will fix it shortly.